linetools/spectra/utils.py

```diff
@@ -35,25 +35,18 @@ def from_spec1d(cls, spec1d):
 
     #### ###############################
     #  Normalize
-    def normalize(self, iconti, verbose=False, no_check=False):
+    def normalize(self, conti, verbose=False, no_check=False):
```

Comment:
The changes in this file have nothing to do with the LineList class, right? (just checking).